protos/feast/storage/Redis.proto

```diff
@@ -1,34 +0,0 @@
-/*
- * Copyright 2019 The Feast Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-syntax = "proto3";
-
-import "feast/types/Field.proto";
-import "feast/types/Value.proto";
-
-package feast.storage;
-
-option java_outer_classname = "RedisProto";
-option java_package = "feast.proto.storage";
-option go_package = "github.com/feast-dev/feast/sdk/go/protos/feast/storage";
-
-message RedisKeyV2 {
```

Comment:
Nervous that this will break things in the java server?

Reply:
https://github.com/feast-dev/feast-java/blob/master/storage/connectors/redis/src/main/java/feast/storage/connectors/redis/common/RedisKeyGenerator.java